test/README.md

```diff
@@ -0,0 +1,95 @@
+# Quick Start
+
+Azure services define their web APIs using [swagger files](https://github.com/azure/azure-rest-api-specs).
+
+To bind those into an SDK, Azure SDK uses [Autorest](https://github.com/Azure/autorest) to parse the swagger definitions into boilerplate networking and serialization code. 
+
+As the Azure SDK has multiple supported languages, Autorest uses plugins for each languages backend. This repository is the C# platform support plugin. 
+
+During autorest execution, the plugin is passed a YAML intermediate representation (IR) of the API definition.
+
+[Example YAML IR](https://github.com/Azure/autorest.csharp/blob/feature/v3/test/TestProjects/AdditionalPropertiesEx/CodeModel.yaml)
+
+The plugin is normally called via JSON RPC but there is a standalone mode that can process the intermediate yaml from disk.
+
+autorest uses a markdown based configuration system (autorest.md) to determine what swagger files to process. 
+
+The C# backend also has an associated csproj next to autorest.md that is processed for customizations and complication of output.
+
+# History
```

Comment:
I think the parts which are not related to tests should go in general read me as a quickstart.